public/data/KendallDoesCoding.json

```
@@ -22,18 +28,19 @@
     {
       "name": "Quiz",
       "url": "https://kendalldoescoding.gq/quiz",
-      "icon": "globe"
+      "icon": "link"
     },
     {
       "name": "Look at my subdomains/projects",
       "url": "https://kendalldoescoding.gq/subdomains",
       "icon": "globe"
-    }
-  ],
+    },
+  ]
   "milestones": [
     {
       "title": "Started Tech Journey",
       "date": "2020"
```

```suggestion
      "date": "2020",
```
Also add `,` at the end of this line.